backend/api/Areas/Property/Mapping/Search/GeoJsonMap.cs

```diff
@@ -42,6 +42,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.ProjectNumbers, src => JsonSerializer.Deserialize<IEnumerable<string>>(src.ProjectNumbers ?? "[]", _serializerOptions))
                 .Map(dest => dest.AddressId, src => src.AddressId)
                 .Map(dest => dest.Address, src => src.Address)
+                .Map(dest => dest.ProjectWorkflow, src => src.ProjectWorkflow)
```

Comment:
This is only tangentially related, but I noticed that the erp/spl search results weren't triggering the correct PIN colour. Returning the workflow code seems like the easiest way to resolve this.